treeherder/webapp/api/performance_data.py

```diff
@@ -889,6 +902,44 @@ def _get_signatures(self, repository_name, framework, interval, no_subtests):
         signatures = self._get_signatures_values(signatures)
         return signatures
 
+    def _create_graph_links(
```

Comment:
@beatrice-acasandrei is there anyone who might know more about the timeRange issue ?

Reply:
No, you'll need to investigate the Compare View UI code to see how the timeRange is decided.